deploy/eck-stack/charts/eck-agent/values.yaml

```diff
@@ -65,8 +65,8 @@ spec:
     #
     # namespace: default
 
-  # Daemonset, or Deployment specification for Agent.
-  # At least one is required of [daemonSet, deployment].
+  # DaemonSet, StatefulSet, or Deployment specification for Agent.
+  # At least one is required of [daemonSet, deployment, statefulSet].
   # No default is currently set until https://github.com/helm/helm/pull/11440 is merged.
```

Comment:
I think it is actually superseded by https://github.com/helm/helm/pull/12162

Reply:
Yes, that's correct, but it's still an open issue see my [comments](https://github.com/helm/helm/issues/12469#issuecomment-1755986299) in another open issue related to the same thing. I'd say we update the link to https://github.com/helm/helm/issues/12469, thoughts?